scripts/entrypoint.sh

```diff
@@ -12,16 +10,16 @@ if [ $# -eq 0 ] ; then
   sh ./scripts/start_server.sh
 elif [ "${1}" = "test" ] ; then
   echo "Run Tests"
-  make test
+  sh ./test/run_tests.sh
```

Comment:
just run the test script directly instead of faffing around with calling it via the makefile